### ISSUE TITLE
Display default image in search results

### ISSUE THREAD

**Issue by ocrasborn** [Signbank/NGT-signbank#147]
* Comes after #145 
* Add a column "Image" before the other columns in the search results
* Show a small thumbnail version of the image so that the line height is only marginally larger. Will require some tweaking depending on the quality of the source image files, but let's start with very small images and see if that makes sense.

**Comment by ocrasborn** [comment]
I've put this issue on hold so that we can first upload an initial set of images, and then test a good solution for downscaling and possible caching them.

**Comment by ocrasborn** [comment]
@Woseseltops , I see you're just working on this so I won't interfere, but (most of) the correct images are waiting to be imported in writable/import_images/, so feel free to import them if you get tired of our Dutch model appearing in every row...

**Comment by Woseseltops** [comment]
Aah that's a nice issue to close the week. Good idea... I think it's done!

Some remarks:
* Used the Boopstrap style for thumbnails by accident.
* I have the urge to click the photo instead of the gloss.

**Comment by ocrasborn** [comment]
@Woseseltops : The height is much better now, but we could perhaps zoom into the sign a bit more by  cropping 5% from the top and 15% from the bottom of each image, keeping the aspect ratio as it is.
Or even better: change the aspect ratio by additionally cropping 15% from the left and right (so without changing the height). So that we'd end up with 20% of the height and 30% of the width of each image.

**Comment by ocrasborn** [comment]
oh, and one more thing: you will implement this for ASL as well, right?

**Comment by Woseseltops** [comment]
It's working for ASL. For the cropping: I spent way too much time investigating whether this was possible, and I think I now found a solution. I will implement it next week, fingers crossed it works!

**Comment by Woseseltops** [comment]
@ocrasborn , I've been so free to ignore the details of your request, and instead played with the various settings until I thought it looked good. I tried to make the signs look (1) as large as possible, (2) without making it too large for the interface and (3) without cropping so much so some signs are no longer visible. You should probably refresh the browser to clear the stylesheet from your cache. Let me know if you want something different.

**Comment by ocrasborn** [comment]
Looks good!

**Comment by Woseseltops** [comment]
Note: this method requires a more or less fixed aspect ratio for the image. I just noticed that all kinds of ratios are used for the ASL Signbank, resulting in some crazy cropping here and there.

**Comment by ocrasborn** [comment]
Hm, yes. For now, it'd be better if we'd apply a different method for ASL: simply scale to have a max. height of X pixels. Would that make things very complicated, making this distinct for NGT vs. ASL?

**Comment by Woseseltops** [comment]
It's overseeable; after some hard thinking, I've decided to add a setting that controls whether images are cropped. Please check whether the result is what we want for ASL-signbank, @ocrasborn

**Comment by ocrasborn** [comment]
Yes, images all look fine now, @Woseseltops . But where does the white border around the images come from? It'd be better to remove that, saving some pixels and thus showing more results on the screen.
(BTW, did you notice the bug in the counting of search results? Total number of signs is messed up.)

**Comment by Woseseltops** [comment]
Done. The white border was a standard bootstrap thingy; we used to have it in NGT Signbank as well, but it is no longer visible because of the cropping.
